inst/templates/issue_template-bug-report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "Open an issue in this repo"
+about: Give us feedback about your issues or unclear documentation
```

Comment:
```suggestion
about: Report an error or unexpected behavior you saw while using this package
```

Again wasn't totally clear from wording about feedback/unclear docs that this was a bug report